cmd/branches.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/branches/create"
+	"github.com/supabase/cli/internal/branches/delete"
+	"github.com/supabase/cli/internal/branches/disable"
+	"github.com/supabase/cli/internal/branches/get"
+	"github.com/supabase/cli/internal/branches/list"
+	"github.com/supabase/cli/internal/branches/update"
+	"github.com/supabase/cli/pkg/api"
+)
+
+var (
+	branchesCmd = &cobra.Command{
+		GroupID: groupManagementAPI,
+		Use:     "branches",
+		Short:   "Manage Supabase preview branches",
+	}
```

Comment:
Maybe make it `--experimental` while the API is still in beta

Reply:
We are now declaring experimental commands in root.cmd so that they are easier to manage.